specs/Modal.spec.js

```diff
@@ -234,25 +244,42 @@ describe('Modal', () => {
     const modal = renderModal({ isOpen: true });
     expect(modal.portal.content.style.position).toEqual(newStyle);
     Modal.defaultStyles.content.position = previousStyle;
+    unmountModal();
+  });
+
+  it('should remove class from body when no modals opened', () => {
+    const expectedBodyClass = 'ReactModal__Body--open';
+    const classList = document.body.classList;
+    renderModal({ isOpen: true });
+    renderModal({ isOpen: true });
+    expect(classList.contains(expectedBodyClass)).toBe(true);
+    unmountModal();
+    expect(classList.contains(expectedBodyClass)).toBe(false);
```

Comment:
I don't see any accessibility and UX problem on stacking modal (regarding that we get to manage the focus and open/exit modals correctly).

Modal should block the UI for a single context or scope (my view). Different from a `window` or `popup`.